### DISCUSSION TITLE
Allow moderators to convert comments to replies

### DISCUSSION THREAD

**Discussion by vkbo** [community/community#49281] [Discussions]
### Select Topic Area

Product Feedback

### Body

The one most frustrating things with the Discussions feature is that the majority of people using Discussions on my project don't notice that Discussions are threaded and tend to add new comments/answers rather than replying when they are actually replying. The Reply box is not as visible and obvious as the designers seem to think it is. As a result, a lot of the threads become disconnected and unreadable if several people participate.

If GitHub don't want to fix their UI (requests for improvements have gone unanswered for years), at least make it possible for moderators to move a comment/answer into another subthread as a reply. I usually tell my users to do this, but not all do, and sometimes I have to manually do it by copy and delete, but then the reply is assigned to me.

**Comment by LindseyB** [comment]
Thanks for your feedback. I've passed it along to the team.
